language/English/strings.po

```diff
@@ -11579,7 +11579,43 @@ msgctxt "#21470"
 msgid "%s [%d]"
 msgstr ""
 
-#empty strings from id 21471 to 21601
+#: system/settings/settings.xml
+msgctxt "#21471"
+msgid "Initially"
+msgstr ""
```

Comment:
@jjd-uk It will select the first unwatched item 'initially' but if the user then focuses on another item then leaves and re-enters the focus will still be on their last selection. Using 'always' upon re-entry will select the first unwatched item again.